src/ChainSafe.Gaming.Marketplace/Dto/MarketplaceItemStatus.cs

```diff
@@ -1,4 +1,8 @@
-﻿namespace ChainSafe.Gaming.Marketplace
+﻿// <copyright file="MarketplaceItemStatus.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
```

Comment:
Remove this placeholder header pls and update from dev. After you update, lint.bat will no longer generate this bs in the files of Marketplace project.

Reply:
done